.travis.yml

```diff
@@ -9,20 +9,24 @@ git:
 os:
   - linux
 env:
-  - V=0.28.0
+  # TRAVIS_TAG should be set as "<artifactSuffix>--<version>"; otherwise publishing will abort
+  global:
+    - BAZEL=0.28.0
+    - COMPILER_CLI_ARTIFACT_ID="play-routes-compiler-cli_$(awk -F '--' '{print $1}' <<< $TRAVIS_TAG)"
```

Comment:
How come there is still a variable named `TRAVIS_TAG`? Isn't it changed to `COMPILER_CLI_ARTIFACT_ID` and `COMPILER_CLI_VERSION`?

Reply:
TRAVIS_TAG is set by travis

Reply:
in `.travis.yml`, custom environment variables are set in the `env` section.
What I'm doing here is setting `COMPILER_CLI_ARTIFACT_ID` to be everything in `TRAVIS_TAG` before the `--` and `COMPILER_CLI_VERSION` to be everything after the `--`